bin/import_fisher.py

```diff
@@ -142,7 +142,7 @@ def _split_wav_and_sentences(data_dir, trans_data, original_data, converted_data
 
             print("splitting {} according to {}".format(wav_files, trans_file))
 
-            origAudios = [wave.open(wav_file, "r") for wav_file in wav_files]
+            origAudios = [librosa.load(wav_file, sr=None, mono=False) for wav_file in wav_files]
```

Comment:
Cool, didn't see that `librosa.load` would use `resampy`, I just used `librosa.load` because all the `resampy` examples did. But I guess I'm just going to leave it as is for now.